components/camel-xmlsecurity/src/test/java/org/apache/camel/component/xmlsecurity/ECDSASignatureTest.java

```diff
@@ -59,8 +59,7 @@ public class ECDSASignatureTest extends CamelTestSupport {
 
     static {
         boolean includeNewLine = true;
-        if (TestSupport.getJavaMajorVersion() >= 9
-                || TestSupport.isJava18_261_later() && !TestSupport.isJavaVendor("Azul")) {
+        if (TestSupport.getJavaMajorVersion() >= 9 || !TestSupport.isJavaVendor("Azul")) {
```

Comment:
It sounds like it is always true since we only support Java 11+, don't you agree?

Reply:
Agreed. Let's simplify this one.

Reply:
For me, it is always true whatever the java vendor since it is a logical OR and we have the first operand that is true